src/test/java/com/google/cloud/resourcemanager/ResourceManagerImplTest.java

```diff
@@ -387,73 +438,210 @@ public void testTestPermissions() {
+      assertEquals(CONSTRAINT_DEFAULT, constraint.getConstraintDefault());
+      assertEquals(DISPLAY_NAME, constraint.getDisplayName());
+      assertEquals(DESCRIPTION, constraint.getDescription());
+      assertEquals(
+          LIST_CONSTRAINT.getSuggestedValue(), constraint.getListConstraint().getSuggestedValue());
+      assertEquals(
+          LIST_CONSTRAINT.getSupportsUnder(), constraint.getListConstraint().getSupportsUnder());
+      assertEquals(VERSION, constraint.getVersion());
+    }
+  }
+
+  @Test
+  public void listAvailableOrgPolicyConstraintsWithException() {
+    String exceptionMessage = "Not Found";
+    Mockito.doThrow(new ResourceManagerException(404, exceptionMessage))
+        .when(resourceManagerMock)
+        .listAvailableOrgPolicyConstraints(RESOURCE);
+    try {
+      resourceManagerMock.listAvailableOrgPolicyConstraints(RESOURCE);
+      fail();
+    } catch (RuntimeException expected) {
+      assertEquals(exceptionMessage, expected.getMessage());
+    }
+  }
+
+  @Test
+  public void testListOrgPolicies() throws IOException {
+    ListResult<OrgPolicy> expectedResult =
+        ListResult.of(CURSOR, ImmutableList.of(ORG_POLICY_INFO.toProtobuf()));
+    Mockito.when(resourceManagerRpcMock.listOrgPolicies(RESOURCE, EMPTY_RPC_OPTIONS))
+        .thenReturn(expectedResult);
+    ListResult<OrgPolicy> results =
+        resourceManagerRpcMock.listOrgPolicies(RESOURCE, EMPTY_RPC_OPTIONS);
+    assertEquals(CURSOR, results.pageToken());
+    for (OrgPolicy orgPolicy : results.results()) {
+      assertEquals(CONSTRAINTS, orgPolicy.getConstraint());
+      assertEquals(ETAG, orgPolicy.getEtag());
+      assertEquals(BOOLEAN_POLICY.getEnforce(), orgPolicy.getBooleanPolicy().getEnforced());
+      assertEquals(LIST_POLICY.getAllValues(), orgPolicy.getListPolicy().getAllValues());
+      assertEquals(LIST_POLICY.getAllowedValues(), orgPolicy.getListPolicy().getAllowedValues());
+      assertEquals(LIST_POLICY.getDeniedValues(), orgPolicy.getListPolicy().getDeniedValues());
+      assertEquals(
+          LIST_POLICY.getInheritFromParent(), orgPolicy.getListPolicy().getInheritFromParent());
+      assertEquals(LIST_POLICY.getSuggestedValue(), orgPolicy.getListPolicy().getSuggestedValue());
+      assertEquals(UPDATE_TIME, orgPolicy.getUpdateTime());
+      assertEquals(VERSION, orgPolicy.getVersion());
+    }
+  }
+
+  @Test
+  public void testListOrgPoliciesWithException() {
+    String exceptionMessage = "Not Found";
+    Mockito.doThrow(new ResourceManagerException(404, exceptionMessage))
+        .when(resourceManagerMock)
+        .listOrgPolicies(RESOURCE);
+    try {
+      resourceManagerMock.listOrgPolicies(RESOURCE);
+      fail();
+    } catch (RuntimeException expected) {
+      assertEquals(exceptionMessage, expected.getMessage());
+    }
+  }
+
+  @Test
+  public void testSetOrgPolicy() {
+    Mockito.when(resourceManagerMock.replaceOrgPolicy(RESOURCE, ORG_POLICY_INFO))
```

Comment:
You've mocked out the method you're trying to test.

also method name, doesn't match method you're testing

Reply:
done